docs/PLUGIN_STATUS.md

```diff
@@ -0,0 +1,80 @@
+* **Experimental** - Early plugins ready for testing but not known to work as intended
+* **Unlabeled** - Shared for reference or extension
+
+Note that plugins can and likely will be demoted if they fall significantly behind best practices of the project. I know that's vague, but it will have to do for this first version :grimacing:
+
+## Plugin Status Matrix
+
+All plugins meet a set of minimum requirements to be included at its status level. Note that some earlier stage plugins *can* have other requirements implemented, but they *must* have all checkboxes to be in the next tier.
+
+| Requirements                   |     Unlabeled      |    Experimental    |      Approved      |     Supported      |
+|:-------------------------------|:------------------:|:------------------:|:------------------:|:------------------:|
+| No naming conflict             | :white_check_mark: | :white_check_mark: | :white_check_mark: | :white_check_mark: |
+| Includes README.md             |                    | :white_check_mark: | :white_check_mark: | :white_check_mark: |
+| Active maintainers             |                    | :white_check_mark: | :white_check_mark: | :white_check_mark: |
+| Release includes binary        |                    |                    | :white_check_mark: | :white_check_mark: |
+| Known to compile               |                    |                    | :white_check_mark: | :white_check_mark: |
+| Known to load in Snap          |                    |                    | :white_check_mark: | :white_check_mark: |
+| Uses Snap Plugin Library       |                    |                    | :white_check_mark: | :white_check_mark: |
+| Reasonable test coverage       |                    |                    | :white_check_mark: | :white_check_mark: |
+| Includes example tasks         |                    |                    | :white_check_mark: | :white_check_mark: |
+| Includes dependency management |                    |                    | :white_check_mark: | :white_check_mark: |
+| Includes CI status             |                    |                    | :white_check_mark: | :white_check_mark: |
+| Includes license               |                    |                    | :white_check_mark: | :white_check_mark: |
+| Support provided by a company  |                    |                    |                    | :white_check_mark: |
+
+## Supported Plugins
+
+These are our premier plugins for the Snap telemetry framework and its users. These are designed to follow all of our recommended practices. Issues are also closely monitored by its supporting company ([read more about that here]()). While companies may support some plugins they contribute to Snap, not all contributions will meet the Supported standard.
+
+We prefer to not have other repositories that overlap with Supported plugins and suggest contributing to the existing version to help keep this list small and effective. Like all plugins, community contribution is welcome.
```

Comment:
I haven't thought of any. Do you have a recommendation @kjlyon? If not, it's worth repeating at the start.